### PR TITLE
Add volume peeling pictures

### PR DESCRIPTION
**Pull request by alvarosan**

### CODE REVIEW

File: results.tex [hunk 1]
```diff
@@ -73,10 +73,13 @@ \subsection{Stochastic Jittering}
   \begin{subfigure}[b]{.5\columnwidth}
     \centering
     \includegraphics[width=\textwidth]{cactus_holland_jittering.png}
-    \caption{With jittering enabled.}
+    \caption{Jittering enabled.}
     \label{fig:jittering_with}
   \end{subfigure}
-  \caption{Stochastic jittering.}
+  \caption{Holland cactus dataset \emph{needCitation} exhibiting wood-grain
```
Comment (alvarosan) [inline results.tex:43]: @cryos ,  do you have the reference for this data?

**Comment by aashish24** [comment]
@alvarosan this is great addition. the geometry rendering looks like not using depth peeling? Or is it just me not seeing it correctly?

File: conclusion.tex [hunk 2]
```diff
@@ -6,35 +6,22 @@ \section{Future Work}
 mapper works as promised by integrating it into existing VTK applications such
 as ParaView and Slicer. Once these tasks are complete, we have some ideas on new
 features we would like to add to this mapper (outlined below). We would also
-like to solicit feedback from folks using the VTK volume mappers. What features
-do you need? Drop us a line at kitware@kitware.com, and let us know.
+like to solicit feedback from the community using the VTK volume mappers. What
+features do you need? Drop us a line at kitware@kitware.com, and let us know.
```
Comment (alvarosan) [inline conclusion.tex]: @aashish24 , what do you think about removing this?

**Comment by alvarosan** [comment]
Yes @aashish24 ,  geometry is definitely being peeled too. What makes it look wrong?

File: conclusion.tex (hunk 2 above)
Comment (sankhesh) [reply conclusion.tex]: +1 to removing solicitation request from the publication.

File: results.tex [hunk 3]
```diff
@@ -5,6 +5,42 @@ \section{Results}
...
+    \centering
+    \includegraphics[width=\columnwidth]{tooth_bottom_enamel.png}
+    \caption{Intermixed \\(rotated).}
+    \label{fig:tooth_bottom_enamel}
+  \end{subfigure}
+  \caption{Industrial CT scan of a human tooth from ~\citep{pfister_tf_bakeoff_2001}.
+    The root and dentin of the tooth are rendered as a volume
+    (~\Autoref{fig:tooth_front}) while an iso-contour of the enamel is rendered
+    as translucent polygonal geometry (~\Autoref{fig:tooth_front_enamel}).
+    ~\Autoref{fig:tooth_bottom} and ~\Autoref{fig:tooth_bottom_enamel} show a
```
Comment (sankhesh) [inline results.tex]: @alvarosan When you add a `~` before `\cite`, `\ref`or `\Autoref`, LaTeX automatically typesets it with a non-breaking space. Typically, you don't require a space before `~` coz you'll end up with two spaces in the document. 

Similarly, you don't need a space after an opening brace `(`.

**TL;DR** 
- Change `and ~\Autoref{...}` to `and~\Autoref{:...}`
- Change `(~\Autoref{...}` to `(\Autoref{...}`

You'll have to make similar changes throughout this diff.

**Review by sankhesh — CHANGES_REQUESTED** [review]

File: results.tex (hunk 3 above)
Comment (alvarosan) [reply results.tex]: Thanks @sankhesh , I did not know this was the case, saw the extra spaces but I forgot to fix them.

**Comment by alvarosan** [comment]
@sankhesh , what do you think about the images?

File: conclusion.tex (hunk 2 above)
Comment (aashish24) [reply conclusion.tex]: I think it probably not best to ask for feedback in paper, I am in favor of removing the kitware@kitware.com reference.

Comment (cryos) [reply conclusion.tex]: I agree, it seems odd, and the reader can easily get to the vtk.org site, and see from there, or use the corresponding author's email address.

File: results.tex (hunk 1 above)
Comment (cryos) [reply results.tex:43]: @alvarosan I don't think that data came from me did it?

Comment (alvarosan) [reply results.tex:43]: I assumed it was, since I got it from kitware@bigmac:~/TestData3D.  Let me know, otherwise I will try to find a reference myself or just use different data.

Comment (cryos) [reply results.tex:43]: I will need to check with the person who gave it to me, I had forgotten about that file. It didn't used to load, and I had been meaning to circle back to it. Let me email him.

**Comment by alvarosan** [comment]
Changed the things you requested @sankhesh .  I will merge this and add two issues for the 2nd iteration of volume peeling images and the jittering image citation.

**Comment by aashish24** [comment]
awesome!
